### PR TITLE
Added 7dof vs 8dof and human pour

### PR DESCRIPTION
**Pull request by akuramshin**
Added content for the two Scenarios of 7dof vs 8dof. Added human pour gif. You can see the video here: [https://akuramshin.github.io/robot-chemist-tamp/](https://akuramshin.github.io/robot-chemist-tamp/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,31 +211,93 @@ <h2 class="title is-3 has-text-centered">Constrained Task and Motion Planning</h
...
+          DoF robot is evaluated in two scenarios:  (1) single step, (2)
+          two steps. In scenario (1), robots find a constrained path
+          with a fixed orientation from initial to final positions that
+          are randomly sampled. Scenario (2) extends the first with an
+          additional intermediate sampled waypoint.
         </p>
       </div>
-      <div class="column is-one-fifth">
-        <img src="static/images/pouring_water.gif" alt="Water pouring"/>
+      <h4 class="subtitle is-4">Scenario 1:</h4>
```
Comment (n-yoshikawa) [inline index.html:230]: I feel "Scenario 1" is not reader-friendly. More explanatory header such as "Scenario 1:  single step" is better. Same for Scenario 2.

File: index.html [hunk 2]
```diff
@@ -211,31 +211,93 @@ <h2 class="title is-3 has-text-centered">Constrained Task and Motion Planning</h
...
+            <!-- Your video here -->
+            <source src="static/videos/8dof_case2.mp4"
+            type="video/mp4">
+          </video>
+          <h4 class="subtitle is-4 has-text-centered">8 Degrees of Freedom</h4>
+        </div>
       </div>
     </div>
+  </div>
+</section>
```
Comment (n-yoshikawa) [inline index.html:270]: Showing the table of 7/8 DoF comparison here may be helpful.

File: index.html [hunk 3]
```diff
@@ -211,31 +211,93 @@ <h2 class="title is-3 has-text-centered">Constrained Task and Motion Planning</h
...
+  <div class="hero-body">
+    <h2 class="title is-3 has-text-centered">Pouring skill</h2>
+    <div class="container">
+      <div class="content">
+        <p>
+        Liquid transfer is one of the essential skills used in a chemistry lab. A pouring skill that intermittently rotates the container is implemented. This pouring strategy compensates the delayed feedback from the scale and outperformed PD controller.  We present and analyze a set of accurate and
+        efficient pouring skills inspired by human motions.
+        </p>
+      </div>
+      <div class="columns">
```
Comment (n-yoshikawa) [inline index.html:283]: The image size is inconsistent with other images, especially the "Constrained Task and Motion Planning" image. Making the pouring image smaller or the constrained motion planning image larger makes the image sizes consistent (I prefer to make the pouring image smaller, at least in the desktop version, but it is up to you).

**Review by kouroshD — APPROVED** [review]

**Comment by akuramshin** [comment]
@n-yoshikawa  I have implemented your suggestions and uploaded to here for preview :[https://akuramshin.github.io/robot-chemist-tamp/]( https://akuramshin.github.io/robot-chemist-tamp/)

**Review by n-yoshikawa — APPROVED** [review]

**Comment by n-yoshikawa** [comment]
Looks good to me. Feel free to merge if you don't have further changes.

**Pull request merged**